playground/PK/RPG Game/ClassCharakter.py

```diff
@@ -0,0 +1,72 @@
+from random import randint
+
+
+class Character:
+    def __init__(self, name, health_points, strenght, crit_chacne, agility, defend, mana_points):
+        self.name = name
+        self.health_points = health_points
+        self.strenght = strenght
+        self.crit_chance = crit_chacne
+        self.agility = agility
+        self.defend = defend
+        self.mana_points = mana_points
+        self.toxicity = 0
+        self.is_alive = True
+        self.backpack = []
+        self.equipment = []
+    #is it ok to add crit strike like that?
+    def get_damage(self, damage):
+        self.health_points = self.health_points - self.reduced_damage(damage)
+        print(f"{self.name} taken {self.reduced_damage(damage)} damage. {self.health_points} health left.")
+
+        if self.health_points <= 0:
+            print(f"{self.name} has died.")
+            self.is_alive = False
+
+    def critical_strike(self, damage, crit_luck):
+        crit_randomise = randint(1,100)
+        print(f"Attacker crit chance is: {crit_randomise}") 
```

Comment:
If I understood correctly, the chance of critical strike is `crit_luck`, not `crit_randomise`. Am I right?